### PR TITLE
Link to VALIDATION.md and CLEANUP.md

### PR DESCRIPTION
**Pull request by bact**
Add language validation/cleanup links to "Get Involved" section in README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ The [Sentence Collector](https://commonvoice.mozilla.org/sentence-collector/) is
 - Is everything working as expected? If not, submit [a new issue](https://github.com/Common-Voice/sentence-collector/issues/new).
 - Review the pending issues in the [project](https://github.com/Common-Voice/sentence-collector/projects/2).
 - Create a [new PR](https://github.com/Common-Voice/sentence-collector/compare) to fix any of the existing issues in the project.
+- To add new language or adjust sentence validation/cleanup: see [VALIDATION.md](https://github.com/common-voice/sentence-collector/blob/main/server/lib/validation/VALIDATION.md) and [CLEANUP.md](https://github.com/common-voice/sentence-collector/blob/main/server/lib/cleanup/CLEANUP.md).
```
Comment (MichaelKohler) [inline README.md:11]: "To add a new language" seems confusing to me, as that only refers to validation and cleanup, not actually adding a new language to contribute to. What do you think of the following?

```suggestion
- To add or adjust validation and cleanup for a language see [VALIDATION.md](https://github.com/common-voice/sentence-collector/blob/main/server/lib/validation/VALIDATION.md) and [CLEANUP.md](https://github.com/common-voice/sentence-collector/blob/main/server/lib/cleanup/CLEANUP.md). Cleanup is optional and should only be used in rare cases.
```

**Review by MichaelKohler — CHANGES_REQUESTED** [review]
Thanks, I like the idea! I've added one comment with a suggestion.

**Comment by bact** [comment]
Thanks for the suggestion. I agree with that.

**Review by MichaelKohler — APPROVED** [review]

**Comment by MichaelKohler** [comment]
Thank you!

**Pull request merged**